packages/mobile/src/screens/favorites-screen/FavoritesScreen.tsx

```diff
@@ -80,6 +81,7 @@ export const FavoritesScreen = () => {
             is_from_favorites: true,
             collection_id: collection.playlist_id.toString()
           }
+          // TODO: include a favorite_created_at timestamp for sorting offline collections
```

Comment:
this is the todo remaining from 1647

Reply:
so we're not currently sorting the favorited collections?

Reply:
that's right. i think less jarring of a ux issue versus playlist contents being wrongly ordered but this is still a todo